src/Store.java

```diff
@@ -0,0 +1,168 @@
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class Store {
+
+    static Bill oneUah = new Bill(1, 0);
+    static Bill twoUah = new Bill(2, 0);
+    static Bill fiveUah = new Bill(5, 0);
+    static Bill tenUah = new Bill(10, 2);
+    static Bill twentyUah = new Bill(20, 0);
+    static Bill hundredUah = new Bill(100, 10);
+    static Bill[] Bills = new Bill[]{oneUah, twoUah, fiveUah, tenUah, twentyUah, hundredUah};
+
+    static ArrayList<String> operationsLog = new ArrayList<String>();
+    static int strategy = 1;
+
+    /**
+     * @param entry string which describe operation with bills in the ATM
+     */
+    private static void addToLog(String entry) {
+        String timeStamp = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(Calendar.getInstance().getTime());
+        operationsLog.add(timeStamp + entry);
+    }
+
+    /**
+     * @param amount - amount of money which user wish to withdraw
+     */
+    public static void giveMoney2(int amount) {
+        Bill[] BillsCopy = new Bill[Bills.length];
```

Comment:
Please never give names to variables which start with a capital letter. Must be `billsCopy`